pom.xml

```diff
@@ -391,7 +391,8 @@ limitations under the License.
                     <execution>
                         <phase>package</phase>
                         <goals>
-                            <goal>jar</goal>
```

Comment:
This modification aims to fix compile errors when compiling with high version maven. The error is `You have to use a classifier to attach supplemental artifacts to the project instead of replacing them.`